test/filterSeatsTest.js

```diff
@@ -45,16 +45,20 @@ describe('Filter seats - Dust helpers', function() {
     done();
   });
 
-  it('should return GSB seats json', function() {
+  it('should return GSB seats ordered by price', function() {
```

Comment:
Yes I wanted to say Golden Silver Bronze seats but probably is better to say seats grouped by quality.